# Message describing the sensor joint angles
Header header
string[] names
float32[] angles